### PR TITLE
Flux GSoD description

### PR DESCRIPTION
**Pull request by dhairyagandhi96**
Ref #745 

cc @MikeInnes

### CODE REVIEW

**Comment by dhairyagandhi96** [comment]
Oops, wrong git Foo

Also not sure why CI is complaining

**Comment by tlienart** [comment]
You can ignore CI it currently only works for collaborators :-)

**Comment by dhairyagandhi96** [comment]
Ah, thanks!

File: jsoc/gsod/projects.md [hunk 1]
```diff
@@ -96,8 +96,24 @@ The [JuliaStats](https://github.com/juliastats) organization hosts packages solv
 
 - ### Reinventing the FluxML Website
 
-  As a part of this, the content on the main landing page https://fluxml.ai can be revamped to highlight some of the interesting progress. The website could use some poilsh with regards to further clarify the messaging without overwhelming a newcomer, while showing off some of the finer grained details to a veteran eye.
+  As a part of this project, the content on the main landing page https://fluxml.ai can be revamped to highlight some of the interesting progress. The website could use some poilsh with regards to further clarify the messaging without overwhelming a newcomer, while showing off some of the finer grained details to a veteran eye.
```
Comment (tlienart) [inline jsoc/gsod/projects.md:99]: use `< ... >` around the URL

Comment (tlienart) [reply jsoc/gsod/projects.md:99]: poilsh -> polish

File: jsoc/gsod/projects.md [hunk 2]
```diff
@@ -96,8 +96,24 @@ The [JuliaStats](https://github.com/juliastats) organization hosts packages solv
...
+  Tools developed by FluxML are widely used by many projects, from Graph Neural Network libraries and Probabilistic Programming libraries to Scientific ML projects and modern large scale Transformer networks. We would like to showoff the flexibility and use cases that we fit in.
```
Comment (tlienart) [inline jsoc/gsod/projects.md:100]: add Markdown links for GNN, PPL, SciML & transformer; `showoff` -> `show off`

File: jsoc/gsod/projects.md [hunk 3]
```diff
@@ -96,8 +96,24 @@ The [JuliaStats](https://github.com/juliastats) organization hosts packages solv
...
+  This could also include cases that help users write custom loss functions, and even putting Flux models behind a web server.
+
+- ### Updated Documentation and DocTests
+
+  Flux documentation needs a lot of rework, since our generic approach to development means there are many general use cases that we support but might be a bit subtle to discover. So in that case, we need to highlight such common cases, and make them known in the documentation.
+  We would like to use doc tests to also increase our coverage of and documentation of many of the common patterns we see in differentiating Julia code.
+
+### Potential Impact
+
+  Flux is an innovative approach to machine learning. This also means that not all the same patterns and assumptions truly hold when translating from a different framework. It also needs a way to communicate a compelling description of how to implement many of the user facing niceties that one might need in the course of completing an ML project. Through this, we want to also find areas of improvement where we could offer a better user experience for our users.
```
Comment (tlienart) [inline jsoc/gsod/projects.md:114]: "a better user experience."

**Pull request merged**